app/assets/javascripts/discourse/app/controllers/preferences/sidebar.js

```diff
@@ -1,13 +1,26 @@
 import Controller from "@ember/controller";
 import { action } from "@ember/object";
 import { tracked } from "@glimmer/tracking";
+import I18n from "I18n";
 
 import { popupAjaxError } from "discourse/lib/ajax-error";
 
 export default class extends Controller {
   @tracked saved = false;
   @tracked selectedSidebarCategories = [];
   @tracked selectedSidebarTagNames = [];
+  @tracked newSidebarTopicDestination =
```

Comment:
Same as https://github.com/discourse/discourse/pull/18594/files#r995348098. I don't think this needs to be a `@tracked` property.